.circleci/test-deploy.yml

```diff
@@ -82,59 +79,57 @@ workflows:
       - integration-test-install-eb-cli:
           context: [CPE-OIDC]
           matrix:
-              parameters:
-                executor: [python, base]
-                # Emtpy string will use latest version, otherwise testing random recent versions
-                cli-version: ["", "3.20.5", "3.20.3", "3.20.2", "3.20.0"]
+            parameters:
+              executor: [python, base]
+              # Emtpy string will use latest version, otherwise testing random recent versions
+              cli_version: ["", "3.20.5", "3.20.3", "3.20.2", "3.20.0"]
           filters: *filters
       - integration-test-deploy-setup:
           context: [CPE-OIDC]
           requires:
-              - integration-test-install-eb-cli
+            - integration-test-install-eb-cli
           filters: *filters
       - aws-elastic-beanstalk/deploy:
           post-steps:
-              - run:
-                  name: Test deployed app
-                  working_directory: ./sample_app
-                  command: curl $(eb status | grep CNAME | cut -d ":" -f 2)
+            - run:
+                name: Test deployed app
+                working_directory: ./sample_app
+                command: curl $(eb status | grep CNAME | cut -d ":" -f 2)
           auth:
             - aws-cli/setup:
                 role-arn: arn:aws:iam::122211685980:role/CPE_ELASTIC_BEANSTALK_TEST
                 role-session-name: OIDC-Test-Session
           context: [CPE-OIDC]
-          environment-name: sample-app-env
-          application-name: sample_app
+          environment_name: sample-app-env
+          application_name: sample_app
           label: test-label
-          app-dir: ./sample_app
+          app_dir: ./sample_app
           requires:
-                - integration-test-deploy-setup
+            - integration-test-deploy-setup
           filters: *filters
       - orb-tools/pack:
           filters: *filters
```

Comment:
You no longer need `orb-tools/pack` running on every build. It should run only upon release; therefore, it must have the same filters as `orb-tools/publish`. See [here](https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/test-deploy.yml#L16) and [here](https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/test-deploy.yml#L41) for the best approach.

Reply:
Done, thank you!